src/gp-model/.gitignore

```diff
@@ -1,5 +1,6 @@
-.env
+.venv
```

Comment:
Hi @wabinyai , we might want to also add the .env file since it also has security details which we do not want to be accessible to the general public on Github. Thanks!